### PR TITLE
fix(mc-http-server): to add referrer policy

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Adds a referrer policy header to the mc-http-server. More details on the interal PR.

### CODE REVIEW

File: packages/mc-http-server/server.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ const fs = require('fs');
 const path = require('path');
 const http = require('http');
 const morgan = require('morgan');
+const hidePoweredBy = require('hide-powered-by');
```
Comment (tdeekens) [inline packages/mc-http-server/server.js]: Just so we use the same package across all services. It's not really needed but I somehow value consistency here.

**Comment by tdeekens** [comment]
> Also, can you include the Feature-Policy as well? It was also mentioned in the document...

On another PR to keep things focussed, okay?

> We should define the header in the headers file, alongside the CSP, etc.

Can you elaborate why? As far as I read the code, through a call chain of 3-5 functions the `headers` are ending up in `createServerIndezMiddleware` which just tdoes `response.setHeader`. Why do you prefer having it from there if it's not configurable?

**Comment by emmenko** [comment]
Using our http-server is not the only way of hosting an app, so the headers should be shipped together.

**Comment by tdeekens** [comment]
Thanks for the explanation. 502d43f

**Pull request merged**